cli/batch/forms.py

```diff
@@ -0,0 +1,35 @@
+"""
+Provide forms for command line interface's batch commands.
+"""
+from marshmallow import (
+    Schema,
+    fields,
+    validate,
+)
+
+from cli.generic.forms.fields import (
+    IdField,
+    IdListField,
+    NodeURLField,
+    ReverseField,
+)
+
+
+class ListBatchesForm(Schema):
+    """
+    List batches validation form.
+    """
+
+    ids = IdListField(allow_none=True, required=False)
```

Comment:
`IdListField` > `BatchIdentifierListField`.

Apply to other cases. For now, we do specific naming like, in the future when we will know then few arguments from different forms could use the same field, we will create a generic one with a generic name.